website/src/pages/tracking.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Layout,
+  SectionLabel,
+  Button,
+  Grid,
+  GridRow,
+  GridItem,
+  GetStartedWithMLflow,
+  SocialWidget,
+} from "../components";
+
+const FakeImage = () => (
+  <div className="w-[600px] h-[400px] bg-black rounded-lg"></div>
+);
+
+export default function QualityMetrics() {
```

Comment:
Similar to above, let's put this in a "classical-ml" subdirectory

```suggestion
export default function Tracking() {
```